src/utils/index.js

```diff
@@ -47,6 +47,11 @@ function getProjectRootPath () {
 function listExtensionsPaths (projectRootPath) {
   const projectRoot = projectRootPath || getProjectRootPath()
   const pkgInfo = readPkgSync({ cwd: path.resolve(projectRoot) })
+  if (!pkgInfo.packageJson.quoti || !pkgInfo.packageJson.quoti.extensions) {
```

Comment:
```suggestion
  if (!pkgInfo.packageJson?.quoti?.extensions?.length) {
```
Isso abrange mais casos e é menor, né? @luizAlbuquerqueJr

Reply:
concordo exceto que não precisava da ? depois do packageJson visto que essa verificação é feita antes no código